### PR TITLE
Assign menu for wrong tab

### PR DESCRIPTION
**Pull request by jonathanKingston**

### CODE REVIEW

**Comment by jonathanKingston** [comment]
Whoops pr for other is in this, sorry.

File: webextension/css/popup.css [hunk 1]
```diff
@@ -133,6 +133,11 @@ table {
   --identity-icon: url("/img/usercontext.svg#chill");
 }
 
+#current-tab [data-identity-icon="default-tab"] {
+  fill: currentColor;
+  background: center center no-repeat url("/img/blank-tab.svg");
```
Comment (groovecoder) [inline webextension/css/popup.css]:
```
webextension/css/popup.css
 138:3  ✖  Expected       order/declaration-block-properties-alphabetical-order
           background to                                                       
           come before                                                         
           fill
```

**Comment by groovecoder** [comment]
Also, I had to restore the `contexts: ["all", "tab"],` in `background.js` to get the context menu to show back up in the tab bar.

File: webextension/css/popup.css (hunk 1 above)
Comment (jonathanKingston) [inline webextension/css/popup.css]: This issue was with the other commit, Pushed there sorry

**Comment by jonathanKingston** [comment]
@groovecoder this isn't the issue. The bug is fixing that when you change the tab you can get the wrong state... in content clicks whilst it is still loading.

**Comment by jonathanKingston** [comment]
Woah how did you get multiple items here in always? Is this normal on mac? @groovecoder

**Comment by groovecoder** [comment]
First time I've ever seen multiple items in that menu.

**Comment by jonathanKingston** [comment]
My mac isn't working to test this, could you run off some console output for me?
I suspect this is removing the addon a few times maybe? I tried replicating for a long time and I am super not convinced this patch caused the double menu.

**Comment by groovecoder** [comment]
I still don't see the original bug fixed on this branch? ... i.e., the "Always open in this Container" context menu option *is checked* until I change tabs or change windows and go back.

>     Open a new container tab (i.e. personal).
>     Go to first page (i.e. twitter.com) and mark it to always open in this container.
>     Go to a second page (i.e. tweetdeck.twitter.com).
>     Right-click to open the context menu and see, the always open in this container menu item is "checked", despite tweetdeck won't open in this container by default, only twitter.
> 

containers-always-open-bug

**Comment by jonathanKingston** [comment]
Hmm this looks like it could be worse on mac perhaps? Again I still don't have a working mac that is easy to test on. Will look into further, it certainly stopped the quick tab switching str to me, perhaps not the typing a new URL I don't think I checked that.

**Comment by groovecoder** [comment]
Are you testing with twitter.com vs. tweetdeck.twitter.com?

**Comment by jonathanKingston** [comment]
@groovecoder no I'm checking for moving quickly between any assigned site and not assigned whilst it is still loading. There are other cases it catches. The twitter/tweetdeck one seems the same though.

I was able to replicate the multiple items issue... again unrelated. It appears reloading the extension in the debugger doesn't delete all context items.

**Comment by jonathanKingston** [comment]
As agreed with @kjozwiak this is easier to manage as one commit as they are all related. #576 is all shield work merged into one

**Pull request closed without merging**